libafl/src/corpus/minimizer.rs

```diff
@@ -133,6 +145,29 @@ where
                 .observers_mut()
                 .post_exec_all(state, &input, &kind)?;
 
+            *state.executions_mut() += 1;
+            let executions = *state.executions();
+
+            curr += 1;
+
+            manager.fire(
+                state,
+                Event::UpdateUserStats {
+                    name: "minimisation exec pass".to_string(),
+                    value: UserStats::Ratio(curr, total),
+                    phantom: Default::default(),
+                },
+            )?;
+
+            manager.fire(
+                state,
+                Event::UpdateExecStats {
```

Comment:
Same question, why so many events?

Reply:
I plan to remove this minimiser in the future regardless... too much trouble to make it compatible with other components. We can point people to the libafl_libfuzzer corpus minimiser and maybe provide an example in fuzzers/